tests/cucumber/features/step_definitions/organizations_steps.coffee

```diff
@@ -40,7 +40,7 @@ do ->
 
     @Then /^I should be on the "([^"]*)" detail page$/, (name, callback) ->
       @browser
-        .waitForVisible('.organization-detail', assert.ifError)
+        .waitForVisible('.organization-detail', 30000, assert.ifError)
```

Comment:
These jobs run in containers on shared machines. I'm trying to anticipate situations where these machines are over-utilized. If the test still fails after waiting for 30 seconds, then that is a clear indication something is wrong.